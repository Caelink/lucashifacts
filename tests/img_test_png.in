#!/bin/bash

export EMAILS=""
export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
export FACTS_SOURCE="$TEST_DIR/img_test_png.lf"
export IMAGE_RARITY=1
export IMG_ASSET_DIR="${RUN_DIR}/tests/img_test_png_img"
