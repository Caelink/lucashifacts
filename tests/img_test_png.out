Subject: lucashi fact of the day
Content-Type: multipart/related; boundary="message"

--message
Content-Type: multipart/alternative; boundary="body"

--body
Content-Type: text/html; charset="UTF-8"

<h1>Hello!</h1>
<p>it is i your dude<br>
thank you for subscribing to<br>
lucashi's lit facts</p>

<p>Did <em>you</em> know that
this lucashi fact is a test fact
</p>
<img src="cid:lucas">

--body--
--message
Content-Type: image/jpeg; name="lucashi.jpg"
Content-Disposition: inline; filename="lucashi.jpg"
Content-Transfer-Encoding: base64
Content-ID: <lucas>
X-Attachment-Id: lucas

iVBORw0KGgoAAAANSUhEUgAAAMgAAADICAIAAAAiOjnJAAAABGdBTUEAALGPC/xhBQAAAiJJREFU
eF7t0IEAAAAAw6D5Ux/khVBhwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIAB
AwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYM
GDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBg
wIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIAB
AwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYM
GDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBg
wIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIAB
AwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYM
GDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDBg
wIABAwYMGDBgwIABAwYMGDBgwIABAwYMGDDwMDDVlwABBWcSrQAAAABJRU5ErkJggg==
--message--
